darwin/objc/runtime.nim

```diff
@@ -10,6 +10,15 @@ type
 
     SEL = distinct pointer
 
+when defined(cpu64):
```

Comment:
I find it easier to just use nim's `int` and `uint` respectively. Those are 1-to-1 matches to `NSInteger` and `NSUInteger`.